src/Symfony/Component/Clock/Clock.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Clock;
+
+/**
+ * A global clock.
+ *
+ * Defined as enum to be final and non-instantiable.
+ */
+enum Clock
+{
+    public static function now(): \DateTimeImmutable
+    {
+        return self::clock()->now();
+    }
+
+    public static function clock(ClockInterface $newClock = null): ClockInterface
```

Comment:
It's not obvious what `Clock::clock()` is about, I needed to look at the implementation. `Clock::new()` instead?

Reply:
`::get()`then?

Reply:
I find `new()` more inline with the method behavior and signature but `get()` works as well. As you prefer!

Reply:
updated!